controllers/nnf_workflow_controller_helpers.go

```diff
@@ -1217,6 +1217,29 @@ func (r *NnfWorkflowReconciler) getNnfNodesFromComputes(ctx context.Context, wor
 	return nnfNodes, nil
 }
 
+func (r *NnfWorkflowReconciler) checkIfContainerJobsStarted(ctx context.Context, workflow *dwsv1alpha1.Workflow) (*result, error) {
+	jobList := &batchv1.JobList{}
+	matchLabels := dwsv1alpha1.MatchingWorkflow(workflow)
```

Comment:
I know it's unlikely we'll ever support more than one container directive, but consider waiting on the specific containers created per the directive, and not _all_ containers for a workflow. That way, if one directive starts all containers OK, but the second directive does not, we know there's a problem with that second directive.